jekyll.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.specification_version = 2 if s.respond_to? :specification_version=
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.rubygems_version = "2.2.2"
-  s.required_ruby_version = ">= 2.0.0"
+  s.required_ruby_version = ">= 2.1.0"
```

Comment:
I'm fine with the documentation updates, but I'm 👎 on this change before Jekyll 4. Our policy on this has been:

- Each major Jekyll version cannot remove support for old Ruby verisons by requiring newer ones in the gemspec. This is considered a breaking change.
- Each minor Jekyll version can deprecate support for an old Ruby version by removing that Ruby version from CI and stating that support has been deprecated in the release blog post.